cvat-canvas/src/typescript/canvasModel.ts

```diff
@@ -649,14 +654,21 @@ export class CanvasModelImpl extends MasterImpl implements CanvasModel {
             this.data.configuration.displayAllText = configuration.displayAllText;
         }
 
-        if (typeof configuration.textFontSize === 'number') {
+        if (typeof configuration.textFontSize === 'number' && configuration.textFontSize >= 8) {
```

Comment:
Maybe const `MIN_TEXT_FONT_SIZE` worths adding. Can be helpful in the future.

Reply:
Yep, maybe

Reply:
Added